src/modules/dashboard/components/ColonyHome/ColonyMeta/ColonyMeta.css

```diff
@@ -13,10 +13,15 @@
 
 .description {
   margin-top: 15px;
+  margin-left: 52px;
+  padding-bottom: 15px;
+  min-height: 75px;
+  border-bottom: 1px solid var(--temp-grey-5);
 }
 
 .dynamicSection {
```

Comment:
The `.dynamicSection` class is not used in `ColonyMeta`

Please clean up the styles file